resources/ajax_forms/getEbscoKbPackageImportForm.php

```diff
@@ -185,8 +185,7 @@
                                     <div class="row">
                                         <div class="col-12 pb-1">
                                             <p>
-                                                You have <?php echo $package->selectedCount; ?> of <?php echo $package->titleCount; ?>
-                                                titles selected in EBSCO Kb.
+                                                <?php echo _('You have '.$package->selectedCount.' of '.$package->titleCount.' titles selected in EBSCO Kb.'); ?>
```

Comment:
This won't behave nicely for translations (because the variables are in the translation string).

Something like this would be better:
`echo sprintf(_('You have %d of %d titles selected in EBSCO Kb'), $package->selectedCount, $package->titleCount);`

Or even better, to display singular or plural form, according to the number of titles:
`echo sprintf(ngettext('You have %d of %d title selected in EBSCO Kb', 'You have %d of %d titles selected in EBSCO Kb', $package->titleCount), $package->selectedCount, $package->titleCount);`